packages/gatsby-cli/src/init-starter.js

```diff
@@ -228,6 +228,14 @@ type InitOptions = {
   rootPath?: string,
 }
 
+const successMessage = path => {
+  report.info(`
+Your new Gatsby site has been successfully bootstrapped. Start developing it by running:
+  $ cd ${path}
```

Comment:
should we include a backticks in case there are spaces in the whole path? or it`s on the response of the user if he use spaces here?


```suggestion
  $ cd \`${path}\`
```

Reply:
How would this show as an output? Like for example if the path is `my-gatsby-starter` ?

Reply:
sorry on command line we use this tick

```suggestion
  $ cd '${path}'
```

Reply:
or replace spaces ` ` with `\ `

```bash
gatsby new 'site with space'
info Creating new site from git: https://github.com/gatsbyjs/gatsby-starter-default.git                                                                                                                 
                                                                                                                                                                                                        
Cloning into 'site with space'...                                                                                                                                                                       
remote: Enumerating objects: 17, done.                                                                                                                                                                  
remote: Counting objects: 100% (17/17), done.                                                                                                                                                           
remote: Compressing objects: 100% (13/13), done.                                                                                                                                                        
remote: Total 2245 (delta 9), reused 12 (delta 4), pack-reused 2228                                                                                                                                     
Receiving objects: 100% (2245/2245), 11.83 MiB | 769.00 KiB/s, done.                                                                                                                                    
success Created starter directory layout                                                                                                                                                                
info Installing packages...                                                                                                                                                                             
                                                                                                                                                                                                        
yarn install v1.17.3                                                                                                                                                                                    
[1/4] 🔍  Resolving packages...                                                                                                                                                                         
[2/4] 🚚  Fetching packages...                                                                                                                                                                          
[3/4] 🔗  Linking dependencies...                                                                                                                                                                       
warning "gatsby > eslint-config-react-app@4.0.1" has incorrect peer dependency "eslint-plugin-flowtype@2.x".                                                                                            
warning "gatsby > @typescript-eslint/eslint-plugin > tsutils@3.17.1" has unmet peer dependency "typescript@>=2.8.0 || >= 3.2.0-dev || >= 3.3.0-dev || >= 3.4.0-dev || >= 3.5.0-dev || >= 3.6.0-dev || >=
 3.6.0-beta || >= 3.7.0-dev || >= 3.7.0-beta".                                                                                                                                                          
[4/4] 🔨  Building fresh packages...                                                                                                                                                                    
success Saved lockfile.                                                                                                                                                                                 
info Initialising git in site with space                                                                                                                                                                
                                                                                                                                                                                                        
info Create initial git commit in site with space   
```

then both works

```
cd site\ with\ space
cd 'site with space'
```

Reply:
But wouldn't it lead to confusion to the user they may actually write those `'` along with the path when following those instructions. Right?

Reply:
I don't know what's the best. Less confusion is with spaces less names is replacing spaces with `\ ` and then technicaly it is still correct if some one do it with space

Let's wait what the core team decides

Reply:
Yeah sure! lets wait for their answer!

Reply:
cc @sidharthachatterjee , @wardpeet